content/redirect.js

```diff
@@ -90,14 +88,62 @@ const resolve = (url) => {
   if (!redirects.size) {
     load();
   }
-  return redirects.get(url.toLowerCase()) || url;
+  return redirects.get(url.toLowerCase()) || resolveFundamental(url).url || url;
 };
 
+function sortTuples([a, b], [c, d]) {
```

Comment:
Pardon my ignorance, but why isn't this inlined into the `shortCuts` function, like the `transit` function is.

Reply:
That's also in #1359 
I had it inline but it really wasn't readable. No strong opinion.

Reply:
Yeah, you're right looking at it now.